docs/user_manual/sequencing_containers.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: page
+title: "15. Sequencing Containers"
+section: 15
+---
+
+A sequencing container holds one or more pools and is loaded onto a sequencer for a sequencing run. These actually have
+different names depending on the platform. Illumina and Oxford Nanopore containers are called flow cells, PacBio
+containers are called 8Pacs, etc. "Sequencing container" is a platform-agnostic term that we use for all of these.
+
+{% assign sub = 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Sequencing Container List" %}
+
+To get to the Sequencing Containers list page, click "Containers" under Sequencing in the Instrument Runs list in the
+menu on the left side of the screen. This list includes all containers that have been created in MISO. The list is
+divided into tabs for the different sequencing platforms. Only platforms for which there are active sequencers or
+existing libraries or pools are included. The toolbar at the top of the table includes many controls for working with
+containers.
+
+There is also a Sequencing Containers list on the Edit Run page which allows you to attach containers to the run. This
+containers list has most of the same controls as on the Sequencing Containers list page, plus extra controls for adding
+and removing containers from the run.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Creating a Container" %}
+
+Containers can be created from the Sequencing Containers list page. Select the tab of the platform for which you would
+like to create a container. From the toolbar at the top of the table, click the "Add Container" button. This button
+will be labelled differently depending on the selected platform. For example, it will be "Add Flow Cell" on the
+Illumina tab.
+
+In the dialog that appears, choose the platform that the container will be run on. Next, choose the sequencing
```

Comment:
I know we use platform for two different purposes, but this is especially confusing here. Maybe sequencer model?

Reply:
Changed to Instrument Model; goes with other PR https://github.com/TGAC/miso-lims/pull/1774 that renames platform to instrument model throughout MISO.